src/ansible_navigator/actions/builder.py

```diff
@@ -0,0 +1,95 @@
+"""Run the builder subcommand."""
+import os
+import shutil
+
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+from ..app import App
+from ..configuration_subsystem import ApplicationConfiguration
+from ..configuration_subsystem.definitions import Constants
+from ..runner import Command
+from . import _actions as actions
+
+
+@actions.register
+class Action(App):
+    """Run the builder subcommand."""
+
+    KEGEX = "^b(?:uilder)?$"
+
+    def __init__(self, args: ApplicationConfiguration):
+        """Initialize the action.
+
+        :param args: The current application configuration.
+        """
+        super().__init__(args=args, logger_name=__name__, name="builder")
+        self._runner: Command
+
+    def run_stdout(self) -> Union[None, int]:
```

Comment:
(educational, optional) Hint: it's the same as
```suggestion
    def run_stdout(self) -> Optional[int]:
```

Reply:
Fixed